marlowe-cli/src/Language/Marlowe/CLI/Command/Test.hs

```diff
@@ -52,7 +52,7 @@ parseTestCommand :: O.Mod O.OptionFields NetworkId
 parseTestCommand network socket=
   O.hsubparser
     $ O.commandGroup "Commands for testing contracts:"
---  <> _scriptsCommand network socket
+    <> _scriptsCommand network socket
```

Comment:
We you can remove the underbar prefix on `_scriptsCommand` so it is just `scriptsCommand` everywhere.